### ISSUE TITLE
Password Bytes and PFX Password Differ from PXEThief

### ISSUE THREAD

**Issue by l33tluigi** [csandker/pxethiefy#6]
Encountered an SCCM PXE server with a blank password on the media file, but the password bytes and PFX password differ from PXEThief. The PFX password from pxethiefy looks to have the same password, but with an additional "f158f" at the end. I only came to realize this after having issues pulling secrets with SharpSCCM using the pxethiefy provided command. Oddly enough, even though PXEThief was able to pull/import the PFX with the password and extract secrets, I'm getting the same error from SharpSCCM that the PFX password is wrong when manually trying to pull secrets. I was going to submit a separate issue for that on the SharpSCCM repo, though I'm not sure if I just have an edge case or what.

![pxethiefy](https://github.com/user-attachments/assets/e0943479-99ba-48ba-ba95-0cf9a1a5f8b6)
![pxethief](https://github.com/user-attachments/assets/16dc054e-591d-435c-afb0-698dbeeb8403)

**Comment by csandker** [comment]
So you're saying pxethiefy showed you that the media file had no password and prompted you the `SharpSCCM` command, but  when using this `SharpSCCM get secrets ...` command it failed ?

So far I had no problems with this, but keep us posted.

**Comment by l33tluigi** [comment]
Yup it fails with the error "Provided password for encoded certificate is not correct". But also pointing out that the password provided in the SharpSCCM command has additional characters tacked on that PXEThief didn't have. I'll have to test this in a lab and see if I get a different result.

**Comment by l33tluigi** [comment]
This issue seems to be popping up for others. An issue was submitted in the SharpSCCM repo https://github.com/Mayyhem/SharpSCCM/issues/60
